fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/Literal.java

```diff
@@ -130,6 +130,11 @@ public String toSql() {
         return toString();
     }
 
+    @Override
+    protected String getExpressionName() {
+        return "literal";
```

Comment:
Same as `Expression.getExpressionName()`?

Reply:
it's the same as above, should be removed.